model/metrics/v8js-metrics.yaml

```diff
@@ -0,0 +1,92 @@
+groups:
+  - id: metric.veightjs.gc.duration
+    type: metric
+    metric_name: v8js.gc.duration
+    brief: "Garbage collection duration."
+    instrument: histogram
+    unit: "s"
+    stability: experimental
+    prefix: v8js.gc
+    attributes:
+      - ref: v8js.gc.type
+        requirement_level: recommended
+
+  - id: metric.veightjs.memory.total
+    type: metric
+    metric_name: v8js.memory.total
+    brief: "Total memory size."
+    instrument: updowncounter
+    unit: "By"
+    stability: experimental
+
+  - id: metric.veightjs.memory.used
+    type: metric
+    metric_name: v8js.memory.used
+    brief: "Used memory size."
+    instrument: updowncounter
+    unit: "By"
+    stability: experimental
+
+  - id: metric.veightjs.heap.total
+    type: metric
+    metric_name: v8js.heap.total
+    brief: "Total heap memory size."
+    instrument: updowncounter
+    unit: "By"
+    stability: experimental
+
+  - id: metric.veightjs.heap.used
+    type: metric
+    metric_name: v8js.heap.used
+    brief: "Used heap memory size."
+    instrument: updowncounter
+    unit: "By"
+    stability: experimental
+
+  - id: metric.veightjs.heap.space_size
+    type: metric
+    metric_name: v8js.heap.space_size
+    brief: "Heap space size."
+    instrument: updowncounter
+    unit: "By"
+    stability: experimental
+    prefix: v8js.heap.space_size
+    attributes:
+      - ref: v8js.heap.space.name
+        requirement_level: recommended
+
+  - id: metric.veightjs.heap.space_used_size
```

Comment:
Hum I wonder if this metric shouldn't be like the JVM one `jvm.buffer.memory.usage`. This is to track the usage of heap space, right? Maybe these can be like `v8js.heap.usage` and `v8js.heap.size`?

Reply:
Would be great to have these links in here somewhere!